python/lsst/afw/geom/python/transform.py

```diff
@@ -46,13 +49,34 @@ def of(self, first):
     if first.getToEndpoint() == self.getFromEndpoint():
         return self._of(first)
     else:
-        raise InvalidParameterError(
+        raise lsst.pex.exceptions.InvalidParameterError(
             "Cannot concatenate %r and %r: endpoints do not match."
             % (first, self))
 
 
+def saveToFile(self, path):
+    """Save this @ref pybind11_transform "Transform" to the specified file
+    """
+    className = type(self).__name__
+    bodyText = self.getFrameSet().show()
+    with open(path, "w") as outFile:
+        outFile.write(className + "\n")
+        outFile.write(bodyText)
```

Comment:
The format *is* human-readable and is an AST standard, and very flexible. Pickle support is a good idea and something I'd like to add later. I suspect pickle support will consist of pickling the AST string (without this extra header).